config/initializers/devise.rb

```diff
@@ -116,6 +116,11 @@
   # Default: false
   config.validate_on_invite = true
 
+  # Auto-login after the user accepts the invite. If this is false,
+  # the user will need to manually log in after accepting the invite.
+  # Default: true
+  config.allow_insecure_sign_in_after_accept = false
+
```

Comment:
Nice!

This makes me wonder 🤔  if we're missing any other configuration options from newer versions of Devise, but that's definitely out-of-scope for this PR!

Reply:
I think it'd be worth regenerating that file just for the improved explanations of some of the config options in the latest version. Anyhow: https://trello.com/c/ARpxcKEA/131-update-signons-devise-and-extensions-config